osu.Game.Rulesets.Catch/Objects/Drawables/DrawableBanana.cs

```diff
@@ -22,6 +23,22 @@ protected override Color4 GetComboColour(IReadOnlyList<Color4> comboColours)
             return colour ??= getBananaColour();
         }
 
+        protected override void UpdateInitialTransforms()
+        {
+            base.UpdateInitialTransforms();
+
+            const float end_scale = 0.6f;
+            const float random_scale_range = 1.6f;
+
+            ScaleContainer.ScaleTo(HitObject.Scale * (end_scale + random_scale_range * RNG.NextSingle()))
+                          .Then().ScaleTo(HitObject.Scale * end_scale, HitObject.TimePreempt);
+
+            const float random_angle_range = 180;
+
+            ScaleContainer.RotateTo(random_angle_range * (RNG.NextSingle() * 2 - 1))
+                          .Then().RotateTo(random_angle_range * (RNG.NextSingle() * 2 - 1), HitObject.TimePreempt);
```

Comment:
Perhaps an opportunity for a (local?) function here in
```csharp
float GetRandomAngle() => random_angle_range * (RNG.NextSingle() * 2 - 1);
```
to avoid duplication but it's a nitpick.